### DISCUSSION TITLE
subdomain

### DISCUSSION THREAD

**Discussion by dawnog** [globaleaks/globaleaks-whistleblowing-software#4169] [General]
I am creating another site for another business - there will be a subdomain. What location (address) should the subdomain direct to? Where can I find this?

**Comment by evilaliv3** [comment]
You should point it to the same IP or CNAME of your globaleaks server

**Comment by dawnog** [comment]
It works! Thank You

pon., 26 sie 2024 o 20:52 Giovanni Pellerano ***@***.***>
napisał(a):
